docs/faq.md

```diff
@@ -218,6 +219,36 @@ For now, the addons you're using in a composed Storybook will not work.
 
 We're working on overcoming this limitation, and soon you'll be able to use them as if you are working with a non-composed Storybook.
 
+## Can I have a Storybook with no local stories?
+
+Storybook does not work unless you define at least one local story.
```

Comment:
I think you need to define what a local story is

Reply:
Sounds good! I'll merge this as is and fix them in an upcoming PR so we can get this in the next release and get some telemetry already, hope that's ok!